doc/user/metrics/README.md

```diff
@@ -0,0 +1,44 @@
+# Monitoring with Prometheus
+
+[Prometheus][prometheus] is an open-source systems monitoring and alerting toolkit. Below is the overview of the different helpers that exist in operator-sdk to help setup metrics in the generated operator.
+
+## Metrics in operator-sdk
+
+The `func ExposeMetricsPort(ctx context.Context, port int32) (*v1.Service, error)` is a helper function that exposes general metrics about the running program. These metrics are inherited from controller-runtime. The metrics are by default served on `:8383/metrics`. A [Service][service] object is created with the metrics port exposed, which can be then accessed by Prometheus. The Service object is [garbage collected][gc] by the owner, which is determined based on the pods owner in which the leader is running in.
+
+To modify the port the metrics are exposed on, change the variable `metricsHost string = ":8383"` variable in the `cmd/manager/main.go` file of the generated operator.
```

Comment:
I would suggest moving the sentence in the above paragraph to the first sentence of this paragraph.

```suggestion
By default, the metrics are served on `0.0.0.0:8383/metrics`. To modify the port the metrics are exposed on, change the `var metricsPort int32 = 8383` variable in the `cmd/manager/main.go` file of the generated operator.
```

Should we mention anything about changing (or not changing) the `metricsHost` variable?

Reply:
> I would suggest moving the sentence in the above paragraph to the first sentence of this paragraph.

Sorry, not sure I understand that. :)